src/controllers/controllerenumerator.h

```diff
@@ -12,16 +12,10 @@ class Controller;
 class ControllerEnumerator : public QObject {
     Q_OBJECT
   public:
-    ControllerEnumerator();
+    ControllerEnumerator() = default;
```

Comment:
fyi putting the autogenerated definition in the `.cpp` is slightly better. That is because with your change the compiler would generate definitions in each file where this is included (just for the linker to remove them again). But I'm just nitpicking nerdsniping